{"name": "test_div_exception_param[100-9-TypeError]", "status": "passed", "parameters": [{"name": "number1", "value": "'100'"}, {"name": "number2", "value": "9"}, {"name": "exceptResult", "value": "<class 'TypeError'>"}], "start": 1586608681218, "stop": 1586608681218, "uuid": "daa6a6a2-28c0-44c6-8166-5e57418a80b3", "historyId": "47e37c2f56b876e8d59a667d2e1c998c", "testCaseId": "6522d78c4e848f5e21c2b2ab9c36ef03", "fullName": "demo01.test_div#test_div_exception_param", "labels": [{"name": "tag", "value": "level_exception"}, {"name": "parentSuite", "value": "demo01"}, {"name": "suite", "value": "test_div"}, {"name": "host", "value": "ubuntu"}, {"name": "thread", "value": "3790-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "demo01.test_div"}]}